datajoint/__init__.py

```diff
@@ -6,18 +6,25 @@
 
 DataJoint is free software under the LGPL License. In addition, we request
 that any use of DataJoint leading to a publication be acknowledged in the publication.
+
+Citations:
```

Comment:
Minor Comment: Maybe that should be renamed in 'Please cite us:'. Otherwise it sounds as if these are citations of datajoint.